packages/@ember/canary-features/index.ts

```diff
@@ -12,19 +12,8 @@ import { assign } from '@ember/polyfills';
   @public
 */
 
-export const DEFAULT_FEATURES = {
-  EMBER_LIBRARIES_ISREGISTERED: null,
-  EMBER_IMPROVED_INSTRUMENTATION: null,
-  EMBER_MODULE_UNIFICATION: false,
-  EMBER_METAL_TRACKED_PROPERTIES: true,
-  EMBER_GLIMMER_FORWARD_MODIFIERS_WITH_SPLATTRIBUTES: true,
-  EMBER_GLIMMER_ANGLE_BRACKET_BUILT_INS: true,
-  EMBER_NATIVE_DECORATOR_SUPPORT: true,
-  EMBER_GLIMMER_FN_HELPER: true,
-  EMBER_CUSTOM_COMPONENT_ARG_PROXY: true,
-  EMBER_FRAMEWORK_OBJECT_OWNER_ARGUMENT: true,
-  EMBER_GLIMMER_SET_COMPONENT_TEMPLATE: null,
-};
+import { default: DEFAULT_FEATURES } from '@ember/canary-features/default-features';
```

Comment:
tsc errors for these lines:
```
$ tsc --noEmit
packages/@ember/canary-features/index.ts:15:10 - error TS1003: Identifier expected.

15 import { default: DEFAULT_FEATURES } from '@ember/canary-features/default-features';
            ~~~~~~~

packages/@ember/canary-features/index.ts:15:17 - error TS1005: ',' expected.

15 import { default: DEFAULT_FEATURES } from '@ember/canary-features/default-features';
```
How can I declare the type while importing? i.e. apart from embedding it into the `default-features.js`

Reply:
So I think I need to add a [declaration](https://www.typescriptlang.org/docs/handbook/declaration-files/by-example.html#objects-with-properties).

Reply:
Ya, I think you can do:

```js
// packages/@ember/canary-features/default-features.d.ts

export default {
  [key: string]: null | boolean;
}
```